lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Track interaction with the student dashboard..
+ */
+
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.dashboard = edx.dashboard || {};
+
+    edx.dashboard.TrackEvents = function() {
+
+        // Emit an event when the "course title link" is clicked.
+        $(".course-title-link").on("click", function (event) {
```

Comment:
@awais786 you can easily locate some of these elements without adding additional classes. For example, you can select all course title links with `$('.course-title > a')`. Instead of adding new classes, please revisit these selectors and use the existing DOM structure where possible.